### PR TITLE
feat: render existing listbox objects

### PR DESCRIPTION
**Pull request by veinfors**
Allow rendering of existing listbox objects (currently it's only possible to render/mount fields and master items which are then created as session objects) by passing in { qId: "generic-object-id" } into mount `embed.field(` function.

--- TODO before merging ---

- [x] Fix and add unit tests
- [x] Regression testing

### CODE REVIEW

File: apis/nucleus/src/components/listbox/ListBox.jsx [hunk 1]
```diff
@@ -192,7 +191,7 @@ export default function ListBox({
     if (loaderRef.current) {
       loaderRef.current.resetloadMoreItemsCache(true);
       // Skip scrollToItem if we are in selections, or if we dont sort by state.
```
Comment (Caele) [inline apis/nucleus/src/components/listbox/ListBox.jsx:193]: Updat the comment as well

File: apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  useEffect(() => {
+    async function fetchObject() {
+      const modelFromStore = modelStore.get(qId);
+      if (modelFromStore) {
+        setModel(modelFromStore);
+      } else {
+        const m = await app.getObject(qId);
+        const key = m ? `${m.id}` : null;
+        modelStore.set(key, m);
```
Comment (johanlahti) [inline apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx:17]: If `key === null` should we still store `m` with key `null`? Throw error instead?

**Review by johanlahti — APPROVED** [review]
Makes code easier to read with constants added and hopefully better performance/fewer requests. 👍

**Review by DanielS-Qlik — APPROVED** [review]

File: apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx (hunk 2 above)
Comment (veinfors) [reply apis/nucleus/src/components/listbox/hooks/useExistingModel.jsx]: I removed the row
`const key = m ? `${m.id}` : null;`

enigma with throw an error if the object isn't found anyway...

**Pull request merged**